### ISSUE TITLE
Icon: Skin icons missing in ebay-icon

### ISSUE THREAD

**Issue by PrashantAshok** [eBay/ebayui-core#249]
# Bug Report

## Description
24 icons from Skin is missing in `ebay-icon` symbols [folder](https://github.com/eBay/ebayui-core/tree/0.6.0/src/components/ebay-icon/symbols). 
E.g. `<ebay-icon type="inline" name="info"/>` doesn't work. 
This is error that shows up:
```
Error: Cannot find module '/node_modules/@ebay/ebayui-core/dist/components/ebay-icon/symbols/info.marko'
```

## Workaround
Add the svg file manually to the app & use it.

**Comment by ianmcburnie** [comment]
@hramakumar Did you see this issue on QA?

**Comment by seangates** [comment]
@ianmcburnie There are quite a few missing. I've logged them in a spreadsheet, and will share with you.

**Comment by PrashantAshok** [comment]
The list:``` ["filter-grid", "filter-single", "filter-list", "home", "info", "messages", "following", "send", "refresh", "add", "sign-out", "user-profile", "flag", "photo-rotate", "photo-crop", "photo-select-all", "photo-select-none", "photo-brightness", "photo-gallery", "photo-gallery-more", "delete", "thumbs-up", "thumbs-up-selected", "thumbs-down", "thumbs-down-selected", "red-laser"]```

**Comment by ianmcburnie** [comment]
0.6.1 branch created in readiness. We should put this fix out in a patch rather than wait for 0.7.0 which will need to go through QA cycle.
